static/app/views/dashboards/manage/utils.tsx

```diff
@@ -3,6 +3,11 @@ import localStorage from 'sentry/utils/localStorage';
 // TODO(nikkikapadia): delete this file and move contents to index.tsx
 export const SHOW_TEMPLATES_KEY = 'dashboards-show-templates';
 
+export const WIDGET_WIDTH = 300;
+export const PADDING = 16;
+export const DEFAULT_NUM_ROWS = 3;
+export const DEFAULT_NUM_WIDGETS = 8;
```

Comment:
Same note as above, these aren't widgets, right, they're dashboard cards? Might be good to prefix all these constants accordingly. `DASHBOARD_CARD_GRID_PADDING`, etc.